src/App.tsx

```diff
@@ -1,10 +1,15 @@
-function App() {
+import '@code-hike/mdx/styles';
+import './css/global.css';
+import './css/custom-ch.css';
+import Example from './components/Example/example.mdx';
 
+function App() {
   return (
-    <>
-      <h1>Reference App</h1>
-      <h1 className="text-3xl font-bold underline">Hello World!</h1>
-    </>
+    <article>
+      <div>
+        <Example />
```

Comment:
should there be a navigation for the reference apps oder how is this handled if there are more?

Reply:
Yeah that's missing atm. Currently I write a ticket and will came up with a proposal :-)

Reply:
See https://reservix.atlassian.net/browse/DST-432